src/filter/index.js

```diff
@@ -1,13 +1,15 @@
 import TurndownService from 'turndown';
 import turndownPluginGithubFlavouredMarkdown from 'joplin-turndown-plugin-gfm';
 import jsdom from 'jsdom';
+import urlToolkit from 'url-toolkit'
+import isRelativeUrl from 'is-relative-url'
```

Comment:
it seems that this package is not useful, `url-toolkit` do nothing if the url is not relative.